app/models/shelf_item.rb

```diff
@@ -5,16 +5,17 @@
 # ShelfList service.
 
 class ShelfItem
-  attr_reader :call_number, :documents, :key
+  attr_reader :call_number, :documents, :key, :classification
   attr_accessor :match, :nearby
 
-  def initialize(call_number:, key:, label: nil, nearby: false)
+  def initialize(call_number:, key:, label: nil, nearby: false, classification: 'lc')
```

Comment:
Same here. I think the only reason we need to know it is for the components. But if it's in the params hash... 🤷

Reply:
removed